docs/controller-skills/runwaymanagement.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Runway Management
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Duty Runways
+Careful selection of the duty runway at an aerodrome is important to ensure safe, expeditious traffic flow. When considering runway configuration, be mindful of the following operational considerations:
+<ol>
+    <li>Wind direction and speed</li>
+    <li>SOP-dictated preferred runway modes</li>
+    <li>Traffic flow and efficiency</li>
+</ol>
+
+### Wind
+Where possible, select the most into wind runway/s. Your chosen runway configuration should have:
+
+|  | Crosswind | Tailwind |
+| ---- | ---- | ---- |
+| Dry Runway | Max 20kt | Max 5kt |
+| Wet Runway | Max 20kt | None |
+
+!!! tip
+    It is good practice to review the TAF when selecting a runway configuration, so that you are aware of any predicted wind shifts and can plan accordingly.
+
+### Preferred Runway Modes
+Most major aerodrome SOP pages include guidance on preferred runway modes, which cater for traffic levels, common approach types, and airspace restrictions. After considering the prevailing winds, have a look at the appropriate SOP page to determine whether a preferred runway mode exists for the current meteorological conditions.
+
+!!! example
+    The [Sydney Aerodrome SOPs](../aerodromes/sydney.md#runway-modes) dictate that the preferred runway mode is SODROPS (runway 34L for arrivals, runway 16L for departures), however a note below indicates the reduced capacity achievable with this mode and recommends PROPS where traffic levels are expected to be high.
+
+### Traffic Flow
+Where the published SOPs don't include guidance on preferred runway modes, or the situation you are faced with differs from those catered for by the SOPs, consider how you can maximise runway efficiency to handle the current traffic scenario. Remember to be mindful of the impact of the current wind & weather conditions, and any operational pressure the chosen configuration might have on any surrounding controllers.
+
+## Approach Types
+The controller in charge of the ATIS must nominate an approach expectation to allow pilots to prepare themselves and their aircraft as required. Since this decision largely affects the TMA controller, it is a good idea to discuss this with them.
+
+Visual approaches should be nominated on the ATIS when the current meteorological conditions (cloud base and visibility) allow for them. You may nominate visual approaches when:
+
+- There are no clouds within 500ft of the MSA (as found on any DAP chart), and
```

Comment:
Highest MSA?

Reply:
> Highest MSA?

I reckon its more about the appropriate MSA, since you might have a super high MSA in a small quadrant but be landing from the other direction, so it won't affect approaching aircraft.

Reply:
fair enough. Perhaps some clarification specifying that it is based on the appropriate MSA then.

Reply:
Amended

Reply:
What about the MSA for missed approach? Does this need to be considered?

Reply:
> What about the MSA for missed approach? Does this need to be considered?

You'd probably know better than us, but I'd assume not, since an aircraft on an instrument approach will follow the published missed which terminates at/above the MSA, and one on a visual approach will likely follow the same missed approach or be given instructions including climb above the MSA for another try. What do you reckon?

Reply:
I don't have enough knowledge on that to answer properly...

Instead of saying "direction from which the approach will commence", say "relevant sector(s) for the relevant approach" or similar. This might not only cover off the missed approach, but also Arcs that cross multiple sectors.

But another thought, should it be 500ft ABV MSA in Class D & 1000ft ABV MSA in Class C

Reason for this is VMC needs to exist at the MSA which differs between airspace categories.

Again though just a thought, don't mind if left how it is.